### ISSUE TITLE
Change sticky terminology to detach and do not make it the default

### ISSUE THREAD

**Issue by georgebutter** [Elkfox/Focus#21]
``` 
 // Detach unless and append to the end of body
  if (this.config.detach) {
    jQuery(document).ready(function() {
      jQuery(target).detach().appendTo('body');
    });
  };
```

**Comment by oscarstranger** [comment]
Transforms at play: https://codepen.io/oscarine/pen/jpNgKX
